lib/swig/solver/solver_module.i

```diff
@@ -12,11 +12,16 @@
 
 %{
 #include "tick/base_model/model.h"
+#include "tick/solver/enums.h"
```

Comment:
We proabably can do 
`%include "tick/solver/enums.h"` so we don't need to put all enum in the swig files

Reply:
sure

Reply:
It fails for some reason - I remember trying something like this before and not doing it because of problems